File: agents/cuqa_agent/tests/fixtures/c/clean.c
/* C fixture: clean.c — no code smells */
#include <stdio.h>

int add(int a, int b) {
    return a + b;
}

int main(void) {
    int result = add(1, 2);
    printf("Result: %d\n", result);
    return 0;
}

File: agents/cuqa_agent/tests/fixtures/python/clean.py
"""A clean Python module with no code smells — used as a baseline."""


def add(a: int, b: int) -> int:
    """Add two integers."""
    return a + b


def subtract(a: int, b: int) -> int:
    """Subtract b from a."""
    return a - b


class Calculator:
    """Simple, clean calculator class."""

    def multiply(self, a: int, b: int) -> int:
        return a * b

    def divide(self, a: float, b: float) -> float:
        if b == 0:
            raise ValueError("Cannot divide by zero")
        return a / b
